src/resources/espnow.cc

```diff
@@ -157,10 +191,10 @@ static void free_datagram(struct DataGram* datagram) {
 
 static void espnow_send_cb(const uint8_t *mac_addr, esp_now_send_status_t status) {
   tx_status = status;
```

Comment:
Which thread calls this? The wifi thread?

Reply:
Yes. On a "high-priority Wi-Fi task".
Will add a comment in a follow-up PR.